src/com/farzaneh/servlet/productServlet.java

```diff
@@ -0,0 +1,91 @@
+package com.farzaneh.servlet;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.farzaneh.exception.ProcessException;
+import com.farzaneh.model.Product;
+import com.farzaneh.model.ProductCategory;
+import com.farzaneh.service.dao.PersonDAO;
+import com.farzaneh.service.dao.ProductDAO;
+
+@WebServlet("/productServlet")
+public class productServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	public productServlet() {
+		super();
+	}
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		String randomId = null;
+		for (Cookie cookie : request.getCookies()) {
+			if (cookie.getName().equals("user")) {
+				randomId = cookie.getValue();
```

Comment:
if you really want to use cookies, whenever you found it you can break;

Reply:
I removed these lines